scripts/dts/edtlib.py

```diff
@@ -142,7 +142,8 @@ class EDT:
       The bindings directory paths passed to __init__()
     """
     def __init__(self, dts, bindings_dirs, warn_file=None,
-                 warn_reg_unit_address_mismatch=True):
+                 warn_reg_unit_address_mismatch=True,
+                 default_prop_types=None):
```

Comment:
This means ever user (kconfiglib, sanitycheck, etc) has to define the same set of default props?  That doesn't make sense to me.